### PR TITLE
Remove submit on change from user select in reservation page

### PR DESCRIPTION
**Pull request by gaetanHerve**

### CODE REVIEW

**Comment by gaetanHerve** [comment]
maybe we should wait... It seems we have an unexpected behaviour with this modification... I have to make some tests

**Comment by gaetanHerve** [comment]
OK. submit on change active on user select was meant to avoid reloading the whole page for just uploading clients list when changing user. I'm gonna use my dynamic forms external here

**Comment by gaetanHerve** [comment]
OK now

File: Modules/booking/View/Bookingdefault/addreservationAction.php [hunk 1]
```diff
@@ -7,6 +7,19 @@
...
     <?php echo $form->htmlOpen() ?>
     <?php echo $form->getHtml($lang, false) ?>
+    <script type="module">
+        import {DynamicForms} from '/externals/pfm/dynamics/dynamicForms.js';
+        let dynamicForms = new DynamicForms();
+        let spaceId = <?php echo $id_space?>;
+        let sourceId = "recipient_id";
+        let targets = [
+            {
+                elementId: "responsible_id",
```
Comment (osallou) [inline Modules/booking/View/Bookingdefault/addreservationAction.php:17]: Source and target identical?

**Comment by osallou** [comment]
And if user (and not manager), user also select client?

File: Modules/booking/View/Bookingdefault/addreservationAction.php (hunk 1 above)
Comment (gaetanHerve) [reply Modules/booking/View/Bookingdefault/addreservationAction.php:17]: euh... "recipient" and "responsible" are not identical :)

**Comment by gaetanHerve** [comment]
> And if user (and not manager), user also select client?

Yes, but only amongst the clients they are related to. If there are several clients for a user, a choice must be done somewhere...
By default, the first of user's clients is selected.

File: Modules/booking/View/Bookingdefault/addreservationAction.php (hunk 1 above)
Comment (osallou) [reply Modules/booking/View/Bookingdefault/addreservationAction.php:17]: Bad reading.... i should rest

**Pull request merged**